lib/tests/2wp.js

```diff
@@ -852,6 +852,56 @@ const execute = (description, getRskHost) => {
 
     });
 
+    it('should do a pegout and round down the weis to satoshis as expected', async () => {
+
+      // Arrange
+
+      // Create a pegin for the sender to ensure there is enough funds to pegout and because this is the natural process
+      const senderRecipientInfo = await createSenderRecipientInfo(rskTxHelper, btcTxHelper);
+      const peginValueInSatoshis = btcToSatoshis(0.5);
+      const btcPeginTxHash = await sendPegin(rskTxHelper, btcTxHelper, senderRecipientInfo.btcSenderAddressInfo, satoshisToBtc(peginValueInSatoshis));
+      await ensurePeginIsRegistered(rskTxHelper, btcPeginTxHash);
```

Comment:
Isn't there a new util function for this?

Reply:
Updated.